src/CabanaPD_Particles.hpp

```diff
@@ -220,6 +220,7 @@ class Particles<MemorySpace, PMB, Dimension>
         auto y = sliceCurrentPosition();
         auto vol = sliceVolume();
         auto nofail = sliceNoFail();
+        auto temp = sliceNoFail();
```

Comment:
"sliceNoFail" should be temperature instead.

Reply:
Fixed.